NEWS

```diff
@@ -1,3 +1,19 @@
+28-Jul-2017 IGNITION v0.17.2
+
+  Bug Fixes
+
+    - Fix failure to create files/directories/links on correct filesystem
+    - Fix issue where legacy force flag didn't force Ignition to decide to not
+      reuse existing filesystem
+    - Fix issue where Ignition would rely on the various mkfs utilities to not
+      overwrite existing filesystems, but some mkfs utilities would
```

Comment:
`Prevent VFAT filesystem creation from unconditionally overwriting existing filesystem`?  We should say `VFAT` at the very least.